src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+
+  &__productImage {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 16px auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: #060b35;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    margin-bottom: 15px;
+  }
+
+  &__starsMenu {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    color: #060b35;
+    font-weight: 400;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+    margin-bottom: 16px;
+  }
+
+  &__textPrice {
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+    font-weight: 400;
+  }
+
+  &__priceValue {
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+    color: #060b35;
+  }
+
+  &__button {
+    background-color: #00acdc;
```

Comment:
Maybe put this color: #00acdc into variable, because you use it at least 6 times